docs/git-comparison.md

```diff
@@ -127,6 +127,11 @@ parent.
           support for pushing from non-Git repos yet)</td>
       <td><code>git push --all [&lt;remote&gt;]</code></td>
     </tr>
+    <tr>
+      <td>Add a remote target to the repo</td>
+      <td><code>jj git remote add &lt;remote&gt; &lt;url&gt;</code></td>
+      <td><code>git remote add &lt;remote&gt; &lt;url&gt;</code></td>
+    </tr>
```

Comment:
This is between two `jj git push` commands. Can you move it one row down?

Reply:
done.

Reply:
Thanks.